@stellar/typescript-wallet-sdk/webpack.config.js

```diff
@@ -32,4 +35,9 @@ module.exports = (env = { NODE: false }) => ({
     path: path.resolve(__dirname, "lib"),
   },
   target: env.NODE ? "node" : "web",
+  plugins: [
```

Comment:
also needed to add this for React environment build

(not sure why wasn't needed before, maybe bc webpack version upgraded for this PR)